src/constants.js

```diff
@@ -16,9 +16,11 @@ export default {
     APPS_UNDER_DEV: "Apps under development",
     FAV_APPS: "Favorite Apps",
     MY_PUBLIC_APPS: "My public apps",
+    FEATURED_APPS: "Featured Apps",
     HPC: "High-Performance Computing",
     BROWSE_ALL_APPS: "Browse All Apps",
     BROWSE_ALL_APPS_ID: "pppppppp-pppp-pppp-pppp-pppppppppppp",
+    FEATURED_APPS_ID: "cccccccc-cccc-cccc-cccc-cccccccccccc",
```

Comment:
Wasn't sure if this should be in a config instead - I'll gladly add it if that's the preference

Reply:
My instinct is to say that it should be in a config, but we already have `BROWSE_ALL_APPS_ID` as a constant. Plus I don't think there's a way to update configs without rebuilding the entire UI right now anyway 🤷